src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * The main service provider for the version 2 of the Widgets.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Events\Views\V2\Views\Widgets\Widget_List_View;
+
+/**
+ * Class Service_Provider
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class Service_Provider extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Variable that holds the name of the widgets being created.
+	 *
+	 * @since TBD
+	 *
+	 * @var array<string>
+	 */
+	private $widgets = [
```

Comment:
This is a comment, not a request for change.
I personally do not like `private` properties when those are not "technical" properties.
If an extending class could reasonably override that property, then I would make that `protected`, instead.
This looks like a property an extending class might reasonably override.

Reply:
Changed to protected.